examples/benchmark/tls_bench.c

```diff
@@ -299,7 +317,7 @@ typedef struct {
 /* Global vars for argument parsing */
 int myoptind = 0;
 char* myoptarg = NULL;
-
+int DoneHandShake = 0;
```

Comment:
Should we wrap the uses of `DoneHandShake` in `#ifdef WOLFSSL_DTLS` since only used by DTLS?

Reply:
Yes, we should wrap.